doc/todo/RELEASE

```diff
@@ -9,6 +9,59 @@ Following shell expansions are used in this document:
  ~elektra-ftp .. where ftp files go to
  ~elektra-doc .. where doc flies go to
 
+----------------------------------------------------------------------
+
+This describes what to do for the automated release:
+
+= Tasks before Release =
+
+== Updates ==
+
+- Update doc/COMPILE.md to reflect actually tested setups
+- Write release notes
+
+== Increment Version Number ==
+
+- CMakeLists.txt
+- ~e/scripts/dev/build-debian-package
```

Comment:
```suggestion
```

If we start building the Debian package on every release and every commit, this script is probably useless? At least from my point of view it can be removed, so that we have one place less with the version number.